employee-management-client/src/app/project-details/project-details.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+    <mat-toolbar>
+        <span class="fill-remaining-space"></span>
```

Comment:
Why have you not added the component in components folder

Reply:
We don't have that structure since starting only and all the components are separate. Maybe in future, we can take it as a refractor/enhancement.